ursaspace/ursa-arcgis-toolbox
# VCM-7569 - Enable Customer Notes

### What's Changed

- New `Customer Notes` input box in the `Analytics` tool in the Toolbox
- This input box allows additional notes to be added to the analytic request
- `Customer Notes` tool tip reads: "Include any additional information that may be useful in fulfilling this request.

---
### To Test
- Run all unit test (see readme)
- Submit an analytic Order and see customer notes includes in the order details
 
![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/54d071ba-982a-4123-b3d3-8b826d55d661)

![image](https://github.com/ursaspace/ursa-arcgis-toolbox/assets/142450704/8cb31037-47b6-4fe2-a977-3cbd881e859a)
